source/encoders/handlers/debug_handler.cpp

```diff
@@ -170,8 +168,6 @@ void debug_handler::get_properties(obs_properties_t*, const AVCodec* codec, AVCo
 					out = to_string(opt->default_val.i64);
 					break;
 				case AV_OPT_TYPE_UINT64:
-					out = to_string(static_cast<uint64_t>(opt->default_val.i64));
-					break;
```

Comment:
**Code Clarity:** UINT64 is not the same as FLAGS, they are simply handled as such here.